sql1.txt

```diff
@@ -6,23 +6,32 @@ SELECT * FROM Products LIMIT 5;
 1. Use an SQL SELECT statement to retrieve the first 10 rows of the Customer table, ordered by CustomerName.
 Paste your SQL statement below:
 
+SELECT * FROM Customers ORDER BY CustomerName LIMIT 10;
+
 
 2. Use an SQL Select statement to retrieve the names and prices for all products that cost less than $20.
 Paste your SQL statement below:
 
+select ProductName, Price from Products where Price < 20;
+
 
 3. Retrieve all employees whose last name starts with C.  Paste your SQL statement below:
 
+select * from Employees where LastName like "C%";
+
 
 4. Retrieve the order ID and customer name for all orders where the customer name starts with A.
 Here you will have to do a join of the Orders table and the Customers table.  Paste your SQL
 statement below.
 
+select Orders.OrderID, Customers.CustomerName from Orders join Customers on Orders.CustomerID = Customers.CustomerID where Customers.CustomerName LIKE 'A%';
+
```

Comment:
This is correct, as an enhancement you can also use a table alias so your query ends up like:
```
ELECT o.OrderID, c.CustomerName 
FROM Orders o 
JOIN Customers c ON o.CustomerID = c.CustomerID 
WHERE c.CustomerName LIKE 'A%';
```